src/hotspot/share/oops/instanceKlass.cpp

```diff
@@ -1015,6 +1015,50 @@ void InstanceKlass::initialize_super_interfaces(TRAPS) {
   }
 }
 
+ResourceHashtable<const InstanceKlass*, OopHandle, 107, ResourceObj::C_HEAP, mtClass>
+      _initialization_error_table;
+
+void InstanceKlass::add_initialization_error(Handle exception, TRAPS) {
+  bool created = false;
```

Comment:
Since this function will never throw, by our new convention, it should not be declared TRAPS, and the thread parameter should not be the last.

Reply:
Ok, I changed it around.